hotfix/2.0.1/create_and_destroy_users.patch

```diff
@@ -1,5 +1,5 @@
---- CreateAndDestoryUsers.coffee
-+++ CreateAndDestoryUsers.coffee
+--- CreateAndDestroyUsers.coffee
```

Comment:
This is a history release. We should rather not touch it. cc @mserranom

Reply:
Correct, it would patch the incorrect file.

Reply:
For the reference: The _live_ file is getting renamed in https://github.com/overleaf/overleaf/pull/823/commits/8e8233ad34c0d4c444bac364afc3362e5ebed8e9#diff-59f27daa0e2da232d251c674d03551b3b1473b723a06d798af57bdb79a2cf7df as well.

Reply:
It's OK to rename the file in `HEAD`, shouldn't cause any issue. In this case, the patch is being applied over an older release with the misspelled name inside.

On a related note, a general rule should be to never touch existing `hotfix/*` files, since they're the actual source for the released images.

Reply:
Thanks. I've updated my branch so if I revisit this project, I should ignore that directory.

I haven't gotten around to teaching my tooling to [spell check filenames](https://github.com/check-spelling/check-spelling/wiki/Feature:-check-filenames-and-paths) by default, so the only way I ran into the misspelled filename was because of the patch. It's been on my list of things to think about, but I haven't reasoned through it yet.

I guess for simplicity I could create a synthetic file that contains a list of the files being spell checked and have that included in the files that are spell-checked.